Files/DataModels/SidebarPinnedModel.cs

```diff
@@ -91,7 +92,7 @@ public async Task ShowHideRecycleBinItemAsync(bool show)
                     var recycleBinItem = new LocationItem
                     {
                         Text = ApplicationData.Current.LocalSettings.Values.Get("RecycleBin_Title", "Recycle Bin"),
-                        Font = Application.Current.Resources["RecycleBinIcons"] as FontFamily,
+                        Font = await CoreApplication.MainView.DispatcherQueue.EnqueueAsync(() => Application.Current.Resources["RecycleBinIcons"] as FontFamily),
```

Comment:
Is the font property needed at all? I think it's a leftover from when we showed a glyph instead of the bitmap icon.